### PR TITLE
docs: Bluetooth: Audio: Add LE audio resources

### PR DESCRIPTION
**Pull request by Thalley**
This commit adds a small overview over resources
related to LE Audio, both Zephyr specific and
general information from the Bluetooth SIG.

### CODE REVIEW

File: doc/connectivity/bluetooth/bluetooth-le-audio-arch.rst [hunk 1]
```diff
@@ -327,3 +327,86 @@ The CSIP implementation supports the following roles
...
+
+The LE audio channel on Discord
+*******************************
+
+Zephyr has a specific Discord channel for LE Audio development, which is open to all.
+Find it here at https://discordapp.com/channels/720317445772017664/1207326649591271434 or simply
+search for `ble-audio` from within Discord.
+Since the `ble-audio` channel is open for all,
+we cannot discuss any specifications that are in development in that channel.
+For discussing that require a Bluetooth SIG membership we refer to the `bluetooth-sig`
```
Comment (kruithofa) [inline doc/connectivity/bluetooth/bluetooth-le-audio-arch.rst:346]:
```suggestion
For discussing issues that require a Bluetooth SIG membership we refer to the `bluetooth-sig`
```

File: doc/connectivity/bluetooth/bluetooth-le-audio-arch.rst [hunk 2]
```diff
@@ -327,3 +327,86 @@ The CSIP implementation supports the following roles
...
+**********************
+
+Anyone who is a Bluetooth SIG member and a Zephyr member can join the weekly meetings where we
+discuss and plan the development of LE Audio in Zephyr. You can find the time of the meetings by
+joining the Bluetooth-sig group at https://lists.zephyrproject.org/g/Bluetooth-sig.
+
+Github project
+**************
+
+LE Audio in Zephyr has it own Github project available at
```
Comment (kruithofa) [inline doc/connectivity/bluetooth/bluetooth-le-audio-arch.rst:359]:
```suggestion
LE Audio in Zephyr has its own Github project available at
```

File: doc/connectivity/bluetooth/bluetooth-le-audio-arch.rst [hunk 3]
```diff
@@ -327,3 +327,86 @@ The CSIP implementation supports the following roles
...
+
+There are many specifications for LE Audio,
+and several of them are still being updated and developed.
+To get an overview of the errata for the LE Audio specifications you can visit
+
+* Generic Audio (GA) errata https://bluetooth.atlassian.net/wiki/spaces/GA/pages/1634402349/GAWG+Errata+Lists
+* Hearing Aid (HA) errata https://bluetooth.atlassian.net/wiki/spaces/HA/pages/1634140216/HA+WG+Errata+List
+* Audio, Telephony and Automotive (ATA) errata https://bluetooth.atlassian.net/wiki/spaces/ATA/pages/1668481034/ATA+Errata+Lists
+
+Access to errata require a Bluetooth SIG membership.
```
Comment (kruithofa) [inline doc/connectivity/bluetooth/bluetooth-le-audio-arch.rst:379]:
```suggestion
Access to errata requires a Bluetooth SIG membership.
```

**Review by kruithofa — CHANGES_REQUESTED** [review]
Some minor typos

**Review by kruithofa — APPROVED** [review]

**Review by larsgk — APPROVED** [review]

**Pull request merged**